### PR TITLE
Fix isolate:false for nested rules with media queries

### PR DESCRIPTION
**Pull request by sapegin**
Not sure how can I test that.

### CODE REVIEW

**Comment by kof** [comment]
Testing: relatively easy, you grab the reset rule like [here](https://github.com/cssinjs/jss-isolate/blob/master/src/index.test.js#L122) and look into selector

**Comment by kof** [comment]
jss-isolate has one big reset rule and its selector contains all other selectors.

**Comment by sapegin** [comment]
Is there a proper way to create `conditional` rule without jss-nested?

File: src/index.test.js [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
 
+import nested from 'jss-nested'
 import isolate from './index'
 
 describe('jss-isolate', () => {
   let jss
 
   beforeEach(() => {
-    jss = create().use(isolate())
+    jss = create().use(isolate(), nested())
```
Comment (kof) [inline src/index.test.js]: I would use additional plugins inside of the specific describe calls in the corresponding beforeEach

**Comment by kof** [comment]
merged

**Pull request merged**